pkgs/by-name/ai/airgorah/package.nix

```diff
@@ -0,0 +1,72 @@
+{
+  rustPlatform,
+  lib,
+  fetchFromGitHub,
+  pkg-config,
+  glib,
+  pango,
+  gdk-pixbuf,
+  graphene,
+  gtk4,
+  copyDesktopItems,
+  makeDesktopItem,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "airgorah";
+  version = "0.7.3";
+
+  src = fetchFromGitHub {
+    owner = "martin-olivier";
+    repo = "airgorah";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-cIb40TKkk3gfy4dTP8WyZqQkRGj5nItaQ3NSfexCUOA=";
+  };
+
+  cargoHash = "sha256-sJFOpxBW6Z8ewfyhJuxZByJNgaEmezw15OQt9KjFaec=";
+
+  nativeBuildInputs = [
+    pkg-config
+    copyDesktopItems
```

Comment:
missing `wrapGAppsHook4`? (Don't say it works, you'll notice when people report a problem like file dialog)